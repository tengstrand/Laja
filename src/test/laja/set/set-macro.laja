
#macro value()
Hi!##
#end

#write "target/test-output/set-macro.out"
  #set (x = value())
  #set (length = value().length())
x = {x}
length = {length}
#end