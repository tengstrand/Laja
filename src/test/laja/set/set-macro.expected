x = Hi!
length = 3
